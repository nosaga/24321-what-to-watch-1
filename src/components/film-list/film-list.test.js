import React from "react";
import renderer from "react-test-renderer";
import FilmList from "./film-list.jsx";

it(`FilmCard renders correctly`, () => {
  const tree = renderer
    .create(<FilmList
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
