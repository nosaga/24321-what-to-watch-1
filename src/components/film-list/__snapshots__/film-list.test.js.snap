// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCard renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <button
    className="small-movie-card__play-btn"
    type="button"
  >
    Play
  </button>
  <div
    className="small-movie-card__image"
  >
    <img
      height="175"
      src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
    />
  </h3>
</article>
`;
